docker_run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+###  SET HOST FILE PATHS HERE. All paths must be absolute!  ###
+HOST_CFG=$(pwd)/rabbitmq-worker.conf
+HOST_LOG=$(pwd)/rabbitmq-worker.log
+HOST_ERR=$(pwd)/rabbitmq-worker.err
+###############################################################
+
+## Automated setting of file paths to be used in the container ##
+# Note: Log and error file paths come from the host config file. All paths must be absolute!
+DOCKER_CFG=/etc/rabbitmq-worker.conf
+# Get log and error file paths from the config file
+DOCKER_LOG=$(grep "^LogFile" $HOST_CFG | sed 's/^LogFile\s*=\s*//')
+DOCKER_ERR=$(grep "^ErrFile" $HOST_CFG | sed 's/^ErrFile\s*=\s*//')
+
+## Confirm that host files exist ##
+touch $HOST_LOG $HOST_ERR 2> /dev/null
+if    [[ ! -f $HOST_LOG ]]
```

Comment:
No problem.  I've been using this style for a long time and like the readability, but I will change to use the more standard style.